rest-api-spec/src/main/resources/rest-api-spec/api/search_application.render_query.json

```diff
@@ -0,0 +1,38 @@
+{
+  "search_application.render_query": {
+    "documentation": {
+      "url": "https://www.elastic.co/guide/en/elasticsearch/reference/master/search-application-render-query.html",
+      "description": "Renders a query for a given rendered metadata for a search application query"
+    },
+    "stability": "beta",
+    "visibility": "public",
+    "headers": {
+      "accept": [
+        "application/json"
+      ],
+      "content_type": [
+        "application/json"
+      ]
+    },
+    "url": {
+      "paths": [
+        {
+          "path": "/_application/search_application/{name}/_render_query",
+          "methods": [
+            "POST"
+          ],
+          "parts": {
+            "name": {
+              "type": "string",
+              "description": "The name of the search application to render the query for"
+            }
+          }
+        }
+      ]
+    },
+    "body": {
+      "description": "Search parameters, consisting of all template parameters",
+      "required": true
```

Comment:
I see, thanks for explaining! 

I think it would be more flexible to allow for not needing a body in both - that would allow for testing default params easier in both circumstances. I think we want that for both - to understand default params, and to render the default query. That would help us in not locking a user into what they can do without need.

Not a blocking issue, just wanted to provide the best experience possible with this API 👍